src/ui/public/state_management/__tests__/state.js

```diff
@@ -259,6 +259,22 @@ describe('State Management', function () {
         expect(fatalStub.firstCall.args[0]).to.be.an(Error);
         expect(fatalStub.firstCall.args[0].message).to.match(/github\.com/);
       });
+
+      it('translateHashToRison should fallback to null if parameter can not be parsed', () => {
```

Comment:
It's the method that's as much close to changed code and also involved into the issue described in PR.